bitshares_pricefeed/pricefeed.py

```diff
@@ -276,7 +282,7 @@ def derive2Markets(self, asset, target_symbol):
                             symbol,
                             target_symbol,
                             float(self.data[interasset][target_symbol][idx]["price"] * ratio["price"]),
-                            float(self.data[interasset][target_symbol][idx]["price"] * ratio["price"]),
+                            float(self.data[interasset][target_symbol][idx]["volume"] * ratio["price"]),
```

Comment:
nice catch!

Reply:
To be fair it's not mine, see #36 . I don't know why @notchxor abandonned his changes.

[Original change cherrypicked](https://github.com/xeroc/bitshares-pricefeed/pull/36/files#diff-a3e6ca7153610d0afb67b00aaac85b1dR280).